artifacts/Dockerfile

```diff
@@ -138,6 +141,8 @@ COPY /src/deployment/docker/id-authentication/* ${ida_sh_path}/
 
 COPY /src/hsm/client.zip ${hsm_client_path}/
 
+COPY /src/hsm/client-21.zip ${hsm_client_path}/
+
```

Comment:
COPY statements to copy images is not added.

Reply:
I didn't understand the above comment, can you please explain the above?

I've taken the changes from https://github.com/mosip/artifactory-ref-impl/blob/release-0.9.0-INJI/artifacts/Dockerfile